unitSI/tests/test_units.py

```diff
@@ -799,137 +747,66 @@ def test_unit_discarding_functions():
     """
     Test functions that discard units.
     """
-    from unitSI.unitsafefunctions import ones_like, zeros_like
 
     values = [3 * mV, np.array([1, 2]) * mV, np.arange(12).reshape(3, 4) * mV]
     for value in values:
         assert_equal(np.sign(value), np.sign(np.asarray(value)))
-        assert_equal(zeros_like(value), np.zeros_like(np.asarray(value)))
-        assert_equal(ones_like(value), np.ones_like(np.asarray(value)))
+        assert_equal(np.zeros_like(value), np.zeros_like(np.asarray(value)))
```

Comment:
This check is a bit misleading, I think. Exceptionally, we allow zero values to compare equal even if the units don't match (`0*mV == 0` is `True`). Here, the test should rather be like the one for `ones_like`, and also test with `have_same_dimensions` that the dimensions are actually preserved.